content/dharma/abstract-data-structures.md

```diff
@@ -0,0 +1,118 @@
+---
+Title: "Introduction to ADTs in JavaScript"
+Description: "An introduction to Abstract Data Types"
+DatePublished: "2022-04-25"
+Categories:
+  - "code-foundations"
+  - "computer-science"
+Tags:
+  - "Data Structures"
+  - "Classes"
+  - "OOP"
+CatalogContent:
+  - "paths/computer-science"
+  - "introduction-to-javascript"
```

Comment:
```suggestion
  - "introduction-to-javascript"
  - "paths/computer-science"
```

If any `"paths/..."` slugs are listed here, they should follow any free "Learn [This]" slugs like `"introduction-to-javascript"`.